documentation/modules/auxiliary/scanner/mssql/mssql_login.md

```diff
@@ -15,6 +15,41 @@ A docker container can be spun up with the following command to test this module
 
 ## Options
 
+### CreateSession
+
+When using the mssql_login module, the CreateSession option can be used to obtain an interactive
+session within the smb instance. Running the following commands with all other options set:
+
+```
+set CreateSession true
+run
+```
+
+Should give you output containing
+
+```
+[*] MSSQL session 1 opened
+```
+
+Which you can interact with using `sessions -i SESSION_NUMBER` or `sessions -1` to interact with the most recently opened session.
+
+```
+msf6 auxiliary(scanner/mssql/mssql_login) > sessions -1
+[*] Starting interaction with 1...
+
+mssql @ 192.168.2.242:1433 (master) > query 'select @@version;'
```

Comment:
What are you're thoughts on having the the help command output here instead, that way it would show users the help command output as well as how to interact with a session.

Reply:
Adding to that, I would maybe add a section on the help command and `query_interactive`. Same as above, showing the help command output for `query_interactive` might be nice.